crmsh/bootstrap.py

```diff
@@ -1423,7 +1423,7 @@ def init_cluster():
     status("Loading initial cluster configuration")
 
     crm_configure_load("update", """
-property cib-bootstrap-options: stonith-enabled=false placement-strategy=balanced
+property cib-bootstrap-options: stonith-enabled=false placement-strategy=default
```

Comment:
Thanks for the change, @liangxin1300! How about we default it just by dropping the placement-strategy setting from here so that the configuration is simplified?

Reply:
OK, changed!